### PR TITLE
#162171008 Configure Authors Haven to use PostgreSQL as DB Engine

### PR DESCRIPTION
**Pull request by akpante3**
#### What does this PR do?
  - it configures postgreSQL as db engine

#### Description of Task to be completed?
  - Removed MongoDB as DB engine
  - Implemented PostgreSQL as DB engine
  - Refactored code from es5 to es6
  - installed Babel to transpile es6 to es5

#### How should this be manually tested?
  - npm run start:dev

#### Any background context you want to provide?
  - N/A

#### What are the relevant pivotal tracker stories?
  - [#162171008](https://www.pivotaltracker.com/story/show/162171008)

#### Screenshots (if appropriate)
  - N/A

#### Questions:
  - N/A

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Chidi3b3r3) [inline .vscode/settings.json]: Please, .gitignore this directory

File: package.json [hunk 2]
```diff
@@ -4,29 +4,29 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start:dev": "nodemon index.js --exec babel-node -- ",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "migration": "sequelize db:migrate"
```
Comment (Chidi3b3r3) [inline package.json]: Name this file `db:migrate` which is more descriptive

File: .vscode/settings.json (hunk 1 above)
Comment (akpante3) [reply .vscode/settings.json]: I have implemented this feedback

File: package.json (hunk 2 above)
Comment (akpante3) [reply package.json]: @Chidi3b3r3 i have implemented this feedback

**Pull request merged**